script/update_deleted_enlistments_for_deleted_projects.rb

```diff
@@ -3,18 +3,23 @@
 raise 'RAILS_ENV is undefined' unless ENV['RAILS_ENV']
 
 require_relative '../config/environment'
+require 'logger'
+$LOG = Logger.new('update_deleted_enlistments_log_file.log')
```

Comment:
This is fine for this script, since it will only be loaded when run, but the `$VARIABLE` syntax is potentially dangerous because it defines a global variable, which, if previously defined would alter that definition. Simply using an initial capital letter, such as "Log" would create a constant and that would generate a warning should something try to reassign a value to it. 

One is well served by creating a class just for the purpose of the script, for example `class UpdateDeletedEnlistmentsForDeletedProjects` and then take advantage of instance variables such as `@log`.

A nice and short example is: https://github.com/blackducksoftware/ohloh-ui/blob/master/script/fix_updated_at_for_topics

It's up to you whether or not you want to refactor the code or not.